public/global.css

```
@@ -140,6 +140,14 @@ a.button:focus {
   padding: .625rem 3.75rem;
 }
 
+* {
+  outline: none;
+}
+
+*:focus {
+  outline: none;
+}
```

Please **never** do this.

```suggestion
*:focus {
  outline: none;
}
```

It's a usability anti-pattern.

Source:

 - https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/
 - https://hacks.mozilla.org/2019/06/indicating-focus-to-improve-accessibility/#content-main:~:text=Never%20remove%20them
 - https://www.washington.edu/accessibility/checklist/focus/#mobile-sidebar:~:text=Avoid%20Overriding%20Browsers%E2%80%99%20Default%20Focus%20Indicator

---

Thanks a lot for bringing this up! I'll definitely make note of that 👍